MusicXML.scm

```diff
@@ -267,8 +266,8 @@
 
       ((NoteEvent)
        (let* ((pitch (ly:music-property m 'pitch))
-              (tie-start (if tie-start-pitches (lset<= equal? (list pitch) tie-start-pitches) #f))
-              (tie-stop (if tie-stop-pitches (lset<= equal? (list pitch) tie-stop-pitches) #f)))
+              (tie-start (if (and tie-start-pitches (member pitch tie-start-pitches)) #t #f))
+              (tie-stop (if (and tie-stop-pitches (member pitch tie-stop-pitches)) #t #f)))
```

Comment:
@rshura Yeah, that's better, thanks!  My sense is that idiomatic Scheme style would just use the result of the `member` call as a truthy value rather than using an explicit `#t` (see: https://www.gnu.org/software/guile/manual/guile.html#Booleans).  In that style this could be:

`(tie-stop (and tie-start-pitches (member pitch tie-start-pitches)))`

But one could argue that having the explicit 'if', #t, and #f is clearer and better.  I'm fine with either style.

Reply:
@PaulMorris that's what I got now, great minds think alike :-)